playbooks/files/rax-maas/plugins/rally_performance.py

```diff
@@ -248,4 +267,5 @@ def main():
     parser = make_parser()
     args = parser.parse_args()
     with maas_common.print_output(print_telegraf=args.telegraf_output):
+        plugin_config = PluginConfig(PLUGIN_CONF)
```

Comment:
It seems odd as plugin_config is used as a kind of global variable, I think it's better to pass it into the main function main(plugin_config), and in turn passed it in send_metrics_to_influxdb(plugin_config), what do you think?

Reply:
Thanks for your feedback.  I've updated the PR as you suggested.